### ISSUE TITLE
v5.4 Not handling original ip/port in BSD 

### ISSUE THREAD

**Issue by philipianpearce** [e2guardian/e2guardian#597]
Help required from BSD developers to get original IP/port when in transparent mode working.

Please see function ConnectionHandler::get_original_ip_port in ConnectionHandler.cpp in branch v5.4.dev

The conditional for BSD needs completing so that the original IP and port is obtained.   Thanks!

See #581 and #545

**Comment by philipianpearce** [comment]
@prlw1  - any chance that you may be able to look at this issue that effects BSD platforms?  Same issue remains in v5.5.

**Comment by prlw1** [comment]
One complication is that NetBSD comes with 3 different firewalls(!) `ipf` (solaris), `pf` (openbsd), `npf` (netbsd). I think the other issues mention `pf`... Would it be possible to bring me up to speed on how to reproduce the problem privately?

**Comment by philipianpearce** [comment]
@prlw1 Yes, happy to do that, but can't locate your email address.   Would you let me know by emailing me directly at  philip at e2bn.org?
